rmarkdown-workflow.Rmd

```diff
@@ -1,53 +1,82 @@
 # R Markdown workflow
```

Comment:
This first line `# R Markdown workflow` needs to be deleted. (I think having it here is what is causing the build to fail.)